server/api/region.go

```diff
@@ -262,14 +262,20 @@ func (h *regionsHandler) CheckRegionsReplicated(w http.ResponseWriter, r *http.R
 	}
 
 	regions := rc.ScanRegions(startKey, endKey, -1)
-	replicated := true
+	state := "REPLICATED"
```

Comment:
If the length of regions is 0, will it return "REPLICATED"?

Reply:
Yes, it is expected.

Reply:
we already have a pending status means that a region peer has log lag. Maybe we can choose another word?

Reply:
I can't come up with one, any suggestion?

Reply:
@disksing What about 'PAUSE'?

Reply:
how about `WAITING`? because the region is in the waiting list.

Reply:
> What is the difference between INPROGRESS and WAITING. Sound like they have a similar meaning. BTW, waiting regions is a cache that may be empty sometimes.


Have used that before, but from @rleungx , can we just decide one? :joy:

Reply:
Then I guess any one is fine.